yarn-project/stdlib/src/interfaces/aztec-node.ts

```diff
@@ -226,7 +202,7 @@ export interface AztecNode
    * @param number - The block number being requested.
    * @returns The requested block.
    */
-  getBlock(number: number): Promise<L2Block | undefined>;
+  getBlock(number: L2BlockNumber): Promise<L2Block | undefined>;
```

Comment:
This was incorrect on master as the actual implementation uses the L2BlockNumber type.